
<launch>

    <group ns="ros_toolkit/plotting">
        <param name="queue_size" value="10"/>
        <param name="slop_time" value="2"/>
        <rosparam file="$(find rostk_plotting)/config/rostk_plotting_config.yaml" command="load" />

        <node pkg="rostk_plotting" type="plotting_main.py" name="rostk_plotting" output="screen">

        </node>

    </group>



</launch>
